app/boards/arm/nrf52840_m2/nrf52840_m2.dts

```diff
@@ -55,10 +55,10 @@
 	compatible = "nordic,nrf-uarte";
 	status = "okay";
 	current-speed = <115200>;
-	tx-pin = <19>;
-	rx-pin = <21>;
-	rts-pin = <23>;
-	cts-pin = <25>;
+	tx-pin = <1>;
```

Comment:
This pin is used for the external 32.768kHz crystal. Wondering if that could affect things.

Reply:
Ah, certainly could. Good catch.